#!/usr/bin/env python

PACKAGE='c2_hw'

import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#           Name       Type      Level Description     Default  Min   Max
gen.add('cmd_timeout', double_t,    0, 'cmd timeout',     20.0,  1,   600.0)

#foo = gen.const('foo', str_t, 'Foo', 'FOO')
#bar = gen.const('bar', str_t, 'Bar', 'BAR')
#baz = gen.const('baz', str_t, 'Baz', 'BAZ')
#strings = gen.enum([foo, bar, baz], 'Strings')
#gen.add('string_param', str_t, SensorLevels.RECONFIGURE_RUNNING,
#        'String param', 'Foo', edit_method = strings)

exit(gen.generate(PACKAGE, 'c2_hw_node', 'CalibrateParam'))
